hazelcast-sql/pom.xml

```diff
@@ -143,8 +141,6 @@
                                 </relocation>
                             </relocations>
                             <transformers>
-                                <transformer
```

Comment:
Why is this removed?

Reply:
This is actually a pretty good question and I will need to dig more into it. There seems to be some problem with the transformer.
Ref: https://github.com/apache/maven-shade-plugin/blob/maven-shade-plugin-3.2.4/src/main/java/org/apache/maven/plugins/shade/resource/ApacheLicenseResourceTransformer.java

When we use the transformer, then no license file is included in the resulting JAR. When we don't use it, then 2 files are added - `LICENSE` and `LICENSE.txt`.
Unfortunatelly, it's not the only difference in the JARs!
The root in the JAR **with** the transformer contains 2 directories and 1 file:
```
com
META-INF
com.hazelcast.org.codehaus.commons.compiler.properties
```

The root in the JAR **without** the transformer contains 4 directories and 2 files:
```
com
io
META-INF
nonapi
com.hazelcast.org.codehaus.commons.compiler.properties
LICENSE-ClassGraph.txt
```

Reply:
I used our own transformer instead - the `HazelcastLicenseResourceTransformer`.